File: CSharpPractice/LeetcodePractice/Leetcode198.cs
public class Solution198 {
    public int Rob(int[] nums) {
        List<int> dp = new List<int>();
        for(int i=0;i<nums.Length+2;i++){dp.Add(0);}
        Console.WriteLine(dp.Count);
        for(int i=2;i<dp.Count;i++)
        {
            dp[i]=Math.Max(dp[i-1],dp[i-2]+nums[i-2]);
        }
        return dp[dp.Count-1];
    }
}

File: JavaScriptPractice/Leetcode198.js
var rob = function(nums) 
{
    let dp= new Array(nums.length+2).fill(0);
    for(let i=2;i<dp.length;i++)
    {
        dp[i]=Math.max(dp[i-1],dp[i-2]+nums[i-2]);
    }
    return dp[dp.length-1];
};